pkg/accounting/bucketusage.go

```diff
@@ -10,11 +10,11 @@ import (
 	"github.com/skyrings/skyring-common/tools/uuid"
 )
 
-// BucketUsage is bucket usage rollup repository
+// BucketUsage is bucket usage tally repository
 type BucketUsage interface {
```

Comment:
You weren't there for this conversation, but I'm pretty sure `bucket_storage_tally` and `bucket_bandwith_tally` are replacing this table. However, I'm wondering... We're still going to need to roll up the `bucket_storage_tally` info somewhere, maybe this table could be it?